osu.Game.Rulesets.Osu/Difficulty/Evaluators/FlashlightEvaluator.cs

```diff
@@ -63,7 +63,7 @@ public static double EvaluateDifficultyOf(DifficultyHitObject current, bool hidd
                         smallDistNerf = Math.Min(1.0, jumpDistance / 75.0);
 
                     // We also want to nerf stacks so that only the first object of the stack is accounted for.
-                    double stackNerf = Math.Min(1.0, (currentObj.LazyJumpDistance / scalingFactor) / 25.0);
+                    double stackNerf = Math.Min(1.0, (currentObj.JumpDistance / scalingFactor) / 25.0);
```

Comment:
Following the logic of `LazyJumpDistance` being renamed to `LazyJumpFromEndDistance`, shouldn't this be using `LazyJumpFromEndDistance`?

Reply:
well, let's not make unrelated changes given that it's already been operating this way thus far

Reply:
okay